firebase/helper.js

```diff
@@ -0,0 +1,46 @@
+import { firebase } from './firebase';
+
+function createUser(name) {
+  firebase
+    .database()
+    .ref("users/") //ADD USER ID
+    .set({
+      name
+      //add longitude and latitude
+    });
+}
+
+function userListener() {
+  return firebase
+    .database()
+    .ref("users/")
+    .on('value', function(snapshot) {
+      console.log(snapshot.val());
+      return (snapshot.val())
+    })
+}
+
+function createPin(title, type, details) {
+  firebase
+    .database()
+    .ref("pins/").push() //ADD USER ID
+    .set({
+      title,
+      type,
+      details
+      //add timestamp, id, some other things.
+    });
+}
```

Comment:
my “adding a pin” has a lot of arguments and it will get larger because we will also be receiving lat, long, etc, so could you make the arguments as objects so that the argument order will not matter?